### DISCUSSION TITLE
Support for Selenium Client 4.11?

### DISCUSSION THREAD

**Discussion by giovannipcruz** [appium/java-client#1985] [Q&A]
Are there any plans to release a Appium Java client that's certified to be compatible with Selenium client v4.11? 

As of Chrome version 114, the Chrome driver team has stopped publishing the Chrome driver releases and metadata using their traditional Chrome driver download repository (https://chromedriver.chromium.org/downloads). As of Chrome version 115, Chrome driver releases can only be discovered programmatically using the abovementioned Chrome for Testing JSON endpoints (https://github.com/GoogleChromeLabs/chrome-for-testing#json-api-endpoints). Selenium v4.11 already supports this new way of Chrome driver discovery. The latest version of the Selenium Manager that comes with Selenium client v4.11 uses the CfT endpoints for Chrome driver management. Moving up to Selenium 4.11.0 is necessary if we want the Selenium manager to continue managing what versiojn of the Chrome driver to use automatically.

**Comment by mykola-mokhnach** [comment]
selenium version is not locked in the client dependencies, so you could use it with any range that is compatible according to [the compatibility matrix](https://github.com/appium/java-client#compatibility-matrix)

**Reply by giovannipcruz** [reply]
@mykola-mokhnach the problem is that Selenium client v4.11 is not listed in the [compatibility matrix](https://github.com/appium/java-client#compatibility-matrix) of the Appium java client. Version 4.10.0 is the latest version of the Selenium client listed in the compatibility matrix. 
<img width="1184" alt="image" src="https://github.com/appium/java-client/assets/68793585/3ecd52aa-8be6-4024-a625-e3f272e2c4a2">
